Code/src/main/java/nl/utwente/viskell/ui/components/ValueBlock.java

```diff
@@ -22,16 +24,17 @@
  */
 public class ValueBlock extends Block {
     /** The value of this ValueBlock. */
-    private StringProperty value;
+    protected StringProperty value;
 
     /** The OutputAnchor of this ValueBlock. */
-    private OutputAnchor output;
+    protected OutputAnchor output;
 
     /** The space containing the output anchor. */
-    @FXML private BorderPane outputSpace;
+    @FXML protected BorderPane outputSpace;
+    @FXML protected Label valueType;
```

Comment:
Missing Javadoc for `valueType` makes it seem it relates to the Javadoc for `outputSpace`.